ajax/libs/polyglot/package.json

```diff
@@ -1,10 +1,30 @@
 {
   "name": "polyglot",
   "filename": "js/jquery.polyglot.language.switcher.min.js",
-  "version": "2.2.0",
+  "version": "2.2",
   "description": "Allows you to easily switch between the languages supported by your website.",
-  "homepage": "http://ixtendo.com/polyglot-language-switcher-jquery-plugin/",
+  "author": "tavyy <octavr@gmail.com>",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:tavyy/Polyglot-Language-Switcher.git"
+  },
   "keywords": [
-    "polyglot"
-  ]
+    "polyglot",
+    "language",
+    "dropdown switcher"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/tavyy/Polyglot-Language-Switcher.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "js/jquery.polyglot*",
+          "css/polyglot*"
```

Comment:
In the version we currently have on cdnjs, the `images` directory was also included. Is that not needed now?